addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -4,19 +4,27 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/icons-material": "^5.15.18",
+    "@mui/material": "^5.15.18",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.6.8",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-scripts": "^5.0.1"
+
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "PUBLIC_URL=/reactWebUI/ BUILD_PATH=./dist react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+
+    "eject": "react-scripts eject",
+    "lint": "eslint"
```

Comment:
This should be wired into the check task, to run in CI.

Reply:
Can u tell me how it can be wired in check task, I'm not really aware about it

Reply:
so we've removed eslint from this PR and now we're adding this eslint code to the build file, sounds confusing.
FYI I have created a new branch namely Duplicate-dep-remove and removed eslint in that branch.

Reply:
thanks! , Updated